### PR TITLE
open62541 1.3.2 (new formula)

### PR DESCRIPTION
**Pull request by paperchalice**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/open62541.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  def install
+    cmake_args = std_cmake_args + %w[
+      -DBUILD_SHARED_LIBS=ON
+      -DUA_ENABLE_DISCOVERY=ON
+      -DUA_ENABLE_HISTORIZING=ON
+      -DUA_ENABLE_JSON_ENCODING=ON
+      -S .
+      -B build
+    ]
+    system "cmake", *cmake_args
```
Comment (carlocab) [inline Formula/open62541.rb:20]:
```suggestion
    cmake_args = %w[
      -DBUILD_SHARED_LIBS=ON
      -DUA_ENABLE_DISCOVERY=ON
      -DUA_ENABLE_HISTORIZING=ON
      -DUA_ENABLE_JSON_ENCODING=ON
    ]
    system "cmake", "-S", ".", "-B", "build", *cmake_args, *std_cmake_args
```

File: Formula/open62541.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    (testpath/"test.c").write <<~C
+      #include <open62541/client_config_default.h>
+      #include <assert.h>
+
+      int main(void) {
+        UA_Client *client = UA_Client_new();
+        assert(client != NULL);
+        return 0;
+      }
+    C
```
Comment (carlocab) [inline Formula/open62541.rb:35]:
```suggestion
    (testpath/"test.c").write <<~EOS
      #include <open62541/client_config_default.h>
      #include <assert.h>

      int main(void) {
        UA_Client *client = UA_Client_new();
        assert(client != NULL);
        return 0;
      }
    EOS
```

**Review by carlocab — COMMENTED** [review]
Let's follow the style of other formulae to make this easier to read.

File: Formula/open62541.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  depends_on "python@3.10" => :build
+
+  def install
+    cmake_args = %w[
+      -DBUILD_SHARED_LIBS=ON
+      -DUA_ENABLE_DISCOVERY=ON
+      -DUA_ENABLE_HISTORIZING=ON
+      -DUA_ENABLE_JSON_ENCODING=ON
+    ]
+    system "cmake", "-S", ".", "-B", "build" , *cmake_args, *std_cmake_args
```
Comment (carlocab) [inline Formula/open62541.rb:18]:
```suggestion
    system "cmake", "-S", ".", "-B", "build", *cmake_args, *std_cmake_args
```

**Review by jonchang — APPROVED** [review]

**Pull request closed without merging**